ref/config/serverConfiguration.adoc

```diff
@@ -221,37 +175,28 @@ operator can be one of `+`, `-`, `*`, `/` for example:
 <variable name="threeagain" value="${six/two}" />
 ----
 
-There are a number of predefined variables:
+The following predefined variables are supported:
 
-* `wlp.install.dir` - the location where the Liberty runtime is installed.
-* `wlp.server.name` - the name of the server.
-* `wlp.user.dir` - the location of the `usr` folder. Defaults to
+* `wlp.install.dir`: the location where the Open Liberty runtime is installed.
```

Comment:
This won't render well because of the format of the code marked things. Recommend sticking with -

Reply:
Updated